lib/pinpoint_supported_countries.rb

```diff
@@ -9,6 +9,11 @@ class PinpointSupportedCountries
   PINPOINT_SMS_URL = 'https://docs.aws.amazon.com/pinpoint/latest/userguide/channels-sms-countries.html'.freeze
   PINPOINT_VOICE_URL = 'https://docs.aws.amazon.com/pinpoint/latest/userguide/channels-voice-countries.html'.freeze
 
+  # The list of countries where we have our sender ID registered
+  SENDER_ID_COUNTRIES = %w[
+    PH
+  ].to_set.freeze
```

Comment:
🤷 idk if source code is the best place to leave this, but seeing as the rest of our configs are in source code, might as well leave it. open to suggestions for better spots!

Reply:
Could have something like a `config/pinpoint_sender_id_countries.yml` ? Or incorporate into `pinpoint_overrides.yml` ?

I don't have strong feelings about it though. Here seems fine too.

Reply:
Going to leave it here for now